docs/Concepts/Pruning.md

```diff
@@ -1,25 +1,46 @@
 ---
-description: Pruning
+description: Node type and pruning
 ---
 
-# Pruning
+# Node type and pruning
+
+Besu supports two node types: _full nodes_ and _archive nodes_.
+
+Full nodes have a complete history of every block and every transaction, all fully validated, and
+can serve the network with any data request. With a full node you can check balances, sign and send
+transactions and look at current Dapp data.
+
+Archive nodes have all of this and they also store the intermediary state of every account and
+contract for every block since the genesis block. With an archive node you can do everything you
+do with a full node, as well as access historical state data.
+
+Archive nodes require significantly more disk space (approximately 3TB) than full nodes
+(approximately 750GB).
+
+To run a full node, enable either:
```

Comment:
I think this explanation needs flipping - if you're using fast sync or pruning, then you have a full node not an archive node.  Maybe moving the _To run an archive node_ paragraph first and then making the second paragraph something like: 

If you enable fast sync, pruning, or both, the node does not store all intermediary state and is a full node.

Reply:
Done.